jwst/pipeline/calwebb_spec2.py

```diff
@@ -276,6 +280,8 @@ def process_exposure_product(
             # Call the cube_build step for IFU data;
             # always create a single cube containing multiple
             # wavelength bands
+            # for MIR_MRS data call the residual fringe correction
```

Comment:
Looks to me like the step is being called up in line 257, so why is the comment down here?

Reply:
Forgot to move comment when I moved where the code was run